runtime/runtime/src/ext.rs

```diff
@@ -600,95 +604,9 @@ pub struct TrieNodesCount {
     pub mem_reads: u64,
 }
 
-// struct TrieAccountingCacheMetrics {
```

Comment:
commit 6: Do we basically lose these prometheus metrics or do we export them somewhere else? (I'm not saying we need to have them)

Reply:
Correct we lose them. I don't believe these metrics are as valuable anymore given that the primary purpose of TAC is no longer acting as a cache, but rather maintaining protocol compatibility. The accesses themselves are much closer in execution cost nowadays whether access for a node happens for the first or a subsequent time due to us using memtries for contract execution.

One of a follow-up tasks I have in mind is to equalize the gas costs of touching nodes for the first vs. subsequent times which would allow to get rid of most of the complexity here. But that's a longer term and not as pressing.